packages/animations/browser/src/warning_helpers.ts

```diff
@@ -36,8 +36,12 @@ export function triggerParsingWarnings(name: string, warnings: string[]): void {
 
 export function pushUnrecognizedPropertiesWarning(warnings: string[], props: string[]): void {
   if (ngDevMode && props.length) {
-    warnings.push(
-        `The provided CSS properties are not recognized properties supported for animations: ${
-            props.join(', ')}`);
+    warnings.push(`The following provided properties are not recognized: ${props.join(', ')}`);
+  }
+}
+
+export function pushNonAnimatablePropertiesWarning(warnings: string[], props: string[]): void {
+  if (props.length) {
+    warnings.push(`The following provided properties are not animatable: ${props.join(', ')}`);
```

Comment:
I'd propose to improve the error message to make it a bit more actionable:
```suggestion
    warnings.push(
      `The following provided properties are not animatable: ${props.join(', ')}. ` +
      `The list of supported animatable properties can be found at https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animated_properties`);
```

I think the error message in the `pushUnrecognizedPropertiesWarning` can also be imprved a bit to mention that the properties are not recognized as valid for animations.

Reply:
In a test below I see that this is a part of a bigger error message, so may be those recommendations can be placed there instead (at the end of the message)?

Reply:
:thinking:

Unfortunately I don't think that the recommendations can be placed in the bigger warning message since that warning message lists the warnings found, which are just generic strings, so the only thing I can think we could do that is by refactoring the warnings to also have a type or something so that for the bigger/final warning we can check the presence of certain type of warnings and show suggestions for them.
This would require some refactoring, I am not completely sure something like that is worth it

Regarding adding the recommendation here, since the warnings are going to be shown in a list format at the end I am afraid this could end up looking a bit off, we could end up with something like:

 ```
  The animation trigger triggerName has built with the following warnings:
   - The following provided properties are not animatable: fontFamily
     The list of supported animatable properties can be found at https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animated_properties
   - The following provided properties are not recognized: testProp
     _another_recommendation_
 ```
It may be ok but it looks a bit weird to me :sweat: 
I would either not put any recommendation at all or put them but in a very concise form as in:
 ```
  The animation trigger triggerName has built with the following warnings:
   - The following provided properties are not animatable: fontFamily
    (see: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animated_properties)
  ...
 ```
 
 At least that's what I think, I am happy to go with whatever you see more fit , what do you think? :slightly_smiling_face:

Reply:
Thanks for additional info.

I'd vote for this option that you proposed:
```
 The animation trigger triggerName has built with the following warnings:
  - The following provided properties are not animatable: fontFamily
   (see: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animated_properties)
 ...
```

Reply:
Added! :slightly_smiling_face::+1: 

![Screenshot at 2022-03-13 22-13-07](https://user-images.githubusercontent.com/61631103/158081634-cc3afdd8-095d-4b02-9bd8-f17ae03abc31.png)

But I must point something which I forgot to mention in my previous comment :sweat: 

If you look at how I changed for the warning you'll see that I've added `\n   (see:...`, those 3 spaces are what makes the second line align nicely in the list, so basically the warning itself "knows" that it will be placed in the list and adds those extra spaces accordingly, I don't think this is the cleanest thing ever, but I don't think it can be done in a different way, what do you think?

___

Also regarding the unrecognized properties, if you want I can add a link for that type of warning too :slightly_smiling_face: , for example https://www.w3.org/Style/CSS/all-properties.en.html looks good, what do you think? (if you want I can do that in the followup PR :slightly_smiling_face:)

Reply:
Thanks for updating the error message, it looks great 👍 

> Also regarding the unrecognized properties, if you want I can add a link for that type of warning too

I don't think we need a link for that error message.